src/js/inject.js

```diff
@@ -228,6 +228,52 @@ const postMessagesListeners = {
     TD.mustaches['status/tweet_single.mustache'] = TD.mustaches['status/tweet_single.mustache'].replace('{{>status/tweet_single_footer}} </div>', '{{>status/tweet_single_footer}} <i class="sprite tweet-dogear"></i> </div>');
     TD.mustaches['status/tweet_detail.mustache'] = TD.mustaches['status/tweet_detail.mustache'].replace('</footer> {{/getMainTweet}}', '</footer> {{/getMainTweet}} <i class="sprite tweet-dogear"></i>');
 
+    // Inject items into the interaction bar
+    if (settings.hotlink_item) {
+      let insertedActions = '';
+      let insertedDetails = '';
+      const actionStache = TD.mustaches['status/tweet_single_actions.mustache'];
+      const detailStache = TD.mustaches['status/tweet_detail_actions.mustache'];
+
+      // Duplicate the 'Like' action
+      const actionPrototype = actionStache.match(/(<li[\s\S]*?li>)/gm)[2];
+      const detailPrototype = detailStache.match(/(<li[\s\S]*?li>)/gm)[2];
+
+      TD.mustaches['text/hotlink_action.mustache'] = TD.mustaches['text/favorite_action.mustache']
+        .replace(/Favorite/g, 'Hotlinked')
+        .replace(/Unlike/g, 'Unlink')
+        .replace(/Like/g, 'Hotlink');
+
+      insertedActions += actionPrototype.replace(/Like/g, 'Hotlink')
+        .replace(/icon-favorite/g, 'icon-link')
+        .replace(/icon-heart/g, 'icon-link')
+        .replace(/Favorited/g, 'Hotlinked')
+        .replace(/favorite/g, 'hotlink')
+        .replace(/href="#"/g, 'href="#" data-btd-action="hotlink-media"')
+        .replace(/tweet-action/g, 'btd-tweet-action tweet-action')
+        .replace(/tweet-action-item/g, 'btd-tweet-action-item tweet-action-item');
+      insertedDetails += detailPrototype.replace(/Like/g, 'Hotlink')
+        .replace(/icon-favorite/g, 'icon-link')
+        .replace(/icon-heart/g, 'icon-link')
+        .replace(/Favorited/g, 'Hotlinked')
+        .replace(/favorite/g, 'hotlink')
+        .replace(/href="#"/g, 'href="#" data-btd-action="hotlink-media"')
+        .replace(/tweet-detail-action/g, 'btd-tweet-detail-action tweet-detail-action')
+        .replace(/tweet-detail-action-item/g, 'btd-tweet-action-item tweet-detail-action-item')
+        .replace(/<li class/, '<li style="width: 20%;" class');
+
+      const insertActionPosition = actionStache.lastIndexOf('<li class="tweet-action-item position-rel');
+      TD.mustaches['status/tweet_single_actions.mustache'] = actionStache.substring(0, insertActionPosition)
+        + insertedActions
+        + actionStache.substring(insertActionPosition);
+
+      const insertDetailPosition = detailStache.lastIndexOf('<li class="tweet-detail-action-item position-rel');
+      TD.mustaches['status/tweet_detail_actions.mustache'] = (detailStache.substring(0, insertDetailPosition)
+      + insertedDetails
+      + detailStache.substring(insertDetailPosition))
+        .replace(/<li class/g, '<li style="width: 20%;" class');
+    }
+
```

Comment:
I'm not a huge fan of how replacing smaller elements in a rapid succession, I feel like it's hard to visualise what the output will actually look like. Couldn't you just do something like this 
```js
mustache.replace('{{_i}}Like{{/i}} </span> </a> </li>', `{{_i}}Like{{/i}} </span> </a> </li>
  {{#tweet.entities}}
  action item
  {{/tweet.entities}}
`);
```

Explanation: it seems like the templates are called from within parent templates that are passed a `tweet` object being...the tweet 😁  so you can easily check if the tweet has entities and thus actually needs the action item or not.